components/PageNavigation.vue

```diff
@@ -272,6 +272,7 @@
       margin: 0 0 0.25rem 0;
       position: relative;
       margin-right: 0;
+      margin-bottom: 0;
```

Comment:
I don't see why this is needed.
If it is, in that case you could just set margin: 0 and remove margin-right and margin-bottom.